package.json

```diff
@@ -28,8 +28,7 @@
   "dependencies": {
     "bs-fetch": "^0.5.2",
     "graphql": "^14.1.1",
-    "urql": "1.5.1",
-    "wonka": "^3.2.2"
```

Comment:
I decided to remove our explicit dependency on `wonka` since it'll be shipped with this version of `urql` and BS can still resolve it just fine. Even though including it appears mostly harmless (it'll get deduped by `yarn`), it just seems like overhead we don't necessarily need. Let me know if I'm wrong here and it's better for us to include it explicitly.